assembly/pom.xml

```diff
@@ -78,7 +78,11 @@
     <!--
       Because we don't shade dependencies anymore, we need to restore Guava to compile scope so
       that the libraries Spark depend on have it available. We'll package the version that Spark
-      uses (14.0.1) which is not the same as Hadoop dependencies, but works.
+      uses which is not the same as Hadoop dependencies, but works.
+
+      TODO: consider removing this. It's only required by a few libraries such as
```

Comment:
Could you file a JIRA and make this an IDed TODO like `TODO(SPARK-XXX)`?

Reply:
will do